rcmd/install.go

```diff
@@ -406,11 +420,13 @@ func InstallPackagePlan(
 				log.WithFields(log.Fields{"binary": iu.BinaryPath, "src": pkg.Path}).Debug(iu.Package)
 
 				if iu.Result.ExitCode != -999 {
+					packagesNeeded = packagesNeeded - 1
 					log.WithFields(log.Fields{
 						"package": iu.Package,
-						"version": pkg.Metadata.Package.Version,
-						"repo":    pkg.Metadata.Config.Repo.Name,
-					}).Info("Successfully Installed")
+						"package version": pkg.Metadata.Package.Version,
```

Comment:
no spaces in log fields - `package_version` etc. Though this is also something we want to maintain consistency on. In this case we know the package, why need to be so explicit and call it package version - can just say `version`. This also should be consistent with the plan() nomenclature

Reply:
So this was a strange one. Log.WithFields lists fields in alphabetical order, and I couldn't find another way to have "packages remaining" listed last. If you're okay with having "remaining" listed at the far left end of the fields, I'll change it back. Otherwise it's going to need to go on its own line or as part of the actual logging statement.

Reply:
far left of the fields is fine. If we need to eventually, we can write our own formatter, https://github.com/sirupsen/logrus/issues/329 For now, really we just need to make it in. It definitely doesn't need to be on its own line/actual logging statement